### ISSUE TITLE
In the E-Invoice and E-Way Bill, the TCS Amount is not getting considered under the Other Charges

### ISSUE THREAD

**Issue by JitendraSAW** [resilient-tech/india-compliance#937]
### Information about bug

Team,

There is an issue getting encountered in the latest versions as the TCS amount is not getting considered under the Other Charges which was previously getting considered but now with the latest versions it is not capturing, though the entry method is the same.

Screenshot-1- Where the TCS is considering in the E-Invoice and E-Way Bill
![image](https://github.com/resilient-tech/india-compliance/assets/80260184/7abdc624-730a-4a7d-87d7-ded2048af2ed)

![image](https://github.com/resilient-tech/india-compliance/assets/80260184/bfcd4b7d-2d3f-4641-83cc-cd7de7bcbcaa)

Screenshot-2- Where the TCS is not considered in the E-Invoice and E-Way Bill.

<img width="917" alt="image" src="https://github.com/resilient-tech/india-compliance/assets/80260184/e545a151-bc6e-48e4-aea7-6a470ab7ecdc">

![image](https://github.com/resilient-tech/india-compliance/assets/80260184/d92fd86e-cf4f-463f-9c69-34ddacf9edf5)

Kindly assist us as this is a most urgent and High Priority. Please update us immediately.

### Version

<img width="434" alt="image" src="https://github.com/resilient-tech/india-compliance/assets/80260184/917bcddf-842b-40d9-b758-8e1b859f791e">

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by ljain112** [comment]
Hello @JitendraSAW ,

Recently, there has been update related to TDS in #789.
There may be an issue with your print format, You may need to update how  `Other Charges` are been fetched in your print format.

**Comment by JitendraSAW** [comment]
Under the current e-invoicing mandate, there is no separate provision for TCS under section 206C(1H). While generating the Invoice Reference Number, TCS included in the invoice value should be included in ‘other charges’, and thus, the invoice value will be reported inclusive of TCS. Thus, automatically in GSTR-1 also, this amount will be included in the invoice value.

Please check the below URL for your info

https://cleartax.in/s/tcs-section-2061h-finance-act-2020-e-invoicing-gst

**Comment by JitendraSAW** [comment]
Implement E-invoicing in India
E-invoicing was made mandatory for all businesses with a revenue of more than Rs. 50 crore starting on April 1st, 2021, as part of the third phase.

Additionally, from 2017–2018 through 2021–2022, enterprises with annual revenue of more than Rs. 20 crores are now required to use electronic invoicing.

According to the most recent update, starting on October 1, 2022, companies with a turnover of more than Rs. 10 crore must produce electronic invoices. There is no specific provision for TCS under section 206C(1H)  of the existing e-invoicing mandate.

TCS should be listed under “other charges” when creating the invoice reference number so that it will be reported as part of the total invoice value. Therefore, this amount will also automatically be included in the invoice value on GSTR-1.

This new TCS clause only applies to receipt-based transactions, not sales. Therefore, the seller of goods is obligated to collect TCS on advances that were later subtracted from the invoice.

As a result, it is advised to collect TCS upon receipt rather than at the time the invoice is issued. Additionally, there will be no impact on e-invoicing if TCS is excluded from the invoice.

IN SHORT IT HAS TO BE ADDED UNDER THE OTHER CHARGES AND SHOULD ALSO BE INCLUDED IN THE TAXABLE VALUE.
Refer the below URL

https://mybillbook.in/s/tax/section-206c-tax-collected-at-source/

**Comment by vorasmit** [comment]
> IN SHORT IT HAS TO BE ADDED UNDER THE OTHER CHARGES AND SHOULD ALSO BE INCLUDED IN THE TAXABLE VALUE.

I think you meant Invoice Value instead of Taxable Value.

> There is no specific provision for TCS under section 206C(1H) of the existing e-invoicing mandate.

Meaning, that the govt really doesn't care about this. The Interpretation you shared comes from the Clear Tax team and it could be different.

We have looked at practices followed by other MNCs (outside of Frappe) as well and there is no consistency in how they implement this.

In conclusion, both could be correct and govt wouldn't penalise for either practice as it doesn't lead to tax evasion.

However, we will revert this to older practice and include TCS to other charges.
